src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -1108,7 +1246,32 @@ std::shared_ptr<Pane> Pane::GetActivePane()
 }
 
 // Method Description:
-// - Gets the TermControl of this pane. If this Pane is not a leaf, this will return nullptr.
+// - Gets the TermControl of this pane. If this Pane is not a leaf but is
+//   focused, this will return the control of the last leaf pane that had focus.
+//   Otherwise, this will return the control of the first child of this pane.
+// Arguments:
+// - <none>
+// Return Value:
+// - nullptr if this Pane is an unfocused parent, otherwise the TermControl of this Pane.
+TermControl Pane::GetLastFocusedTerminalControl()

```

Comment:
Hmm, do we still use this externally? Seems like it might be a potential footgun. Like, presumably, if someone's calling this, they don't know to do the `for(auto control = pane.control() in panes if pane.isLeaf()):` thing

Reply:
Yes, the tab/page attempts to focus the last control whenever anything changes, so this is redirected through the tab as `GetActiveTerminalControl`